go-app-ussd_popi_user_data.js

```diff
@@ -226,32 +183,23 @@ go.app = function() {
         });
 
         // OPTIONS MENU
-        self.add('state_all_options_view', function(name) {
+        self.states.add('state_all_options_view', function(name) {
             return new ChoiceState(name, {
                 question: $('What would you like to do?'),
                 choices: [
                     new Choice('state_view', $('See my personal info')),
-                    new Choice('state_view_sms', $('Send my personal info by sms')),
                     new Choice('state_change_data', $('Change my info')),
                     new Choice('state_confirm_delete', $('Request to delete my info')),
                 ],
                 next: function(choice) {
-                    if (choice.value === 'state_view_sms') {
-                        return self
-                        .send_data_as_sms()
-                        .then(function() {
-                            return 'state_view_sms';
-                        });
-                    }else{
-                        return choice.value;
-                    }
+                    return choice.value;
```

Comment:
We can probably now change this to a MenuState and remove this logic: http://vumi-jssandbox-toolkit.readthedocs.io/en/master/states/reference.html#MenuState